bluepyemodel/emodel_pipeline/emodel_settings.py

```diff
@@ -257,12 +268,50 @@ def __init__(
                 and EPSP protocols be run and plotted.
                 Should be True only for pyramidal cells,
                 since it depends on the presence of apical dendrite.
+            plot_IV_curves (bool): during the plotting, should peak voltage and voltage_deflection
```

Comment:
It would be nice if pipeline_settings entries were clearly mentioned. e.g. the "IV_curve_prot_name" is case insensitive. You can even give examples of complex settings which require multiple entries. e.g. you can use the one below:

> "plot_IV_curves":true,
> "plot_FI_curve_comparison":true,
> "plot_traces_comparison":true,
> "run_plot_custom_sinspec":false,
> "IV_curve_prot_name":"iv",
> "FI_curve_prot_name":"idrest",
> "plot_phase_plot":true,
> "phase_plot_settings":{"prot_names":["idrest"], 
>                         "relative_amp":150,
>                         "amp_window":3},
> "sinespec_settings":{"amp": 100,
>                         "threshold_based": true},
> "custom_bluepyefe_cells_pklpath":"./figures/cADpyr_L5TPC/efeatures_extraction/cells.pkl",
> "custom_bluepyefe_protocols_pklpath":"./figures/cADpyr_L5TPC/efeatures_extraction/protocols.pkl"